### DISCUSSION TITLE
Email address registered to different building?

### DISCUSSION THREAD

**Discussion by RealMrCactus** [community/community#137069] [GitHub Education]
### Hi everyone,

So I'm at the high-school right now and the email/website is registered to the Freshmen building would this cause my application to get declined even though they both are apart of the same district? I submitted a dated transcript too for it.

**Comment by imxaander** [comment]
As long as you have these [requirements](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student#requirements) submitted, your application is not going to be declined.

**Reply by RealMrCactus** [reply]
> As long as you have these [requirements](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student#requirements) submitted, your application is not going to be declined.

![image](https://github.com/user-attachments/assets/7b92c396-43a0-4b72-b89a-873def9d55bb)
My email is verified, i can't get a physical replica.
